.github/ISSUE_TEMPLATE/webtestplan.md

```diff
@@ -1029,3 +1050,23 @@ Add the following to enable read access to trusted clusters
   - [ ] Clean the Application Support dir for Connect. Start the latest stable version of the app.
         Open every possible document. Close the app. Start the current alpha. Reopen the tabs. Verify that
         the app was able to reopen the tabs without any errors.
+
+[^cloud-tenant-pre-release-deployments]: Instructions for deploying a custom release to a cloud staging tenant: https://github.com/gravitational/teleport.e/blob/master/dev-deploy.md
+
+[^cloudshell-prerelease-cdn]: The AWS integration enrollment wizards will have you run scripts in AWS cloudshell that try to download a teleport tarball from our usual CDN.
```

Comment:
ya this part is really annoying. could we fix it?

Reply:
we could try to detect pre-release builds and use the staging cdn instead